src/App.js

```diff
@@ -24,11 +24,13 @@ export class App extends Component{
       isLoading: true,
       isOver: false,
       hasGenerator: null,
+      hasWord: false,
       userName: this.props.userName || 'guest',
       rooms: this.props.rooms || [],
       numOnline: 'calculating the number of',
       attemptsLeft: 0,
-      playerNames: []
+      playerNames: [],
+      winners:['--REDACTED--']
```

Comment:
What is the reason for `'--REDACTED--'`? Not complainin', just askin'.

Reply:
A somewhat humorous way to error - test 😆